.github/workflows/publish-release-type.yaml

```diff
@@ -43,3 +45,21 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.githubToken }}
         run: |
           ./build/scripts/do-release-candidate.sh
+      - name: slack on success
```

Comment:
I like how you refactored it to use it for both releases and RCs :) is it possible to use this workflow template for slack notifications for the build-all-distros job as well?

Reply:
I didn't have a choice :grin: the release/RC workflows just invoke this workflow, only place it can be added! I think its not really feasible for build-all-distros